assume/strategies/advanced_orders.py

```diff
@@ -138,7 +138,7 @@ def calculate_bids(
 
             # Flex-bid price formulation
             if op_time <= -unit.min_down_time or op_time > 0:
-                bid_quantity_flex = max_power[start] - bid_quantity_inflex
+                bid_quantity_flex = max_pwr - bid_quantity_inflex
```

Comment:
Why shorten the variable name? I feel this is not as self explanatory

Reply:
max_power is an array, we cannot reuse the name again. we would need to rename max_power to max_power_array in the first place or use a different name here

Reply:
I mean this is just one way of iterating over a list. we can of course agree to rename the lists and iterate over them and call the variables something else. Please make a suggestion what makes more sense from your point of view

Reply:
I have changed the naming everywhere to max_power_values and max power in the iteration. this should avoid any confusions